ydb/public/lib/ydb_cli/dump/file/files.cpp

```diff
@@ -0,0 +1,60 @@
+#include "files.h"
+
+#include <ydb/public/lib/ydb_cli/dump/util/util.h>
+
+#include <contrib/libs/protobuf/src/google/protobuf/text_format.h>
+
+#include <util/generic/string.h>
+#include <util/system/file.h>
+
+namespace NYdb::NBackup::NFiles {
+
+enum EFilesType {
+    SCHEME = 0,
+    PERMISSIONS,
+    CHANGEFEED_DESCRIPTION,
+    TOPIC_DESCRIPTION,
+    INCOMPLETE_DATA,
+    INCOMPLETE,
+    EMPTY
```

Comment:
```suggestion
    EMPTY,
```

Чтобы при расширении списка не нужно было исправлять эту строчку.